tests/ui/circulation/test_actions_extend.py

```
@@ -231,21 +232,27 @@ def test_extend_on_item_on_loan_with_no_requests(
+    # midnight CET (UTC+1) when tests run after 23:00 UTC and produce
+    # different library-local calendar days.
     params = {
         "transaction_location_pid": loc_public_martigny.pid,
         "transaction_user_pid": librarian_martigny.pid,
     }
-    item, actions = item.extend_loan(**params)
-    assert item.status == ItemStatus.ON_LOAN
-    extended_loan = Loan.get_record_by_pid(initial_loan.pid)
+    frozen_now = datetime.now(timezone.utc)
+    with freeze_time(frozen_now):
+        item, actions = item.extend_loan(**params)
```

Ruff (0.15.1)

[warning] Unpacked variable `actions` is never used

Prefix it with an underscore or any other dummy variable pattern

(RUF059)